lib/commands/new/buildsystems/general/integration-test-runner/cypress.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const ProjectItem = require('../../../../../project-item').ProjectItem;
+
+module.exports = function(project) {
+  const cypressResourceFolder = 'test/cypress';
+  const isUsingTypescript = project.model.transpiler.id === 'typescript';
+
+  if (isUsingTypescript) {
+    project.addToDevDependencies(
+      '@cypress/webpack-preprocessor'
+    );
+  }
+
+  const testDirResources = [
+    ProjectItem.directory('fixtures').add(
+      ProjectItem.resource('example.json', `${cypressResourceFolder}/fixtures/example.json`)
+    ),
+    ProjectItem.directory('integration').add(
+      ProjectItem.resource('app.e2e.ext', `${cypressResourceFolder}/integration/app.e2e.ext`, project.model.transpiler)
+    ),
+    ProjectItem.directory('plugins').add(
+      ProjectItem.resource('index.js', `${cypressResourceFolder}/plugins/index.ext`, project.model.transpiler)
+    ),
+    ProjectItem.directory('screenshots'),
+    ProjectItem.directory('support').add(
+      ProjectItem.resource('index.js', `${cypressResourceFolder}/support/index.js`),
+      ProjectItem.resource('commands.ext', `${cypressResourceFolder}/support/commands.ext`, project.model.transpiler)
+    ),
+    ProjectItem.directory('videos')
+  ];
+
+  if (isUsingTypescript) {
+    testDirResources.push(ProjectItem.resource('tsconfig.json', `${cypressResourceFolder}/tsconfig.json`));
+  }
+
+  project.addToTasks(
+    ProjectItem.resource('tsconfig.json', 'content/tsconfig.template.json')
```

Comment:
Is this a mistake?

Reply:
I'm assuming you are referring to the tsconfig always being added? I should have made this conditional.

Reply:
The top level tsconfig is handled by transpilers/typescript. I don't think you want copy it again here (even with condition).

Reply:
I see, you are right. Removed handling the tsconfig at this level.